packages/fields/src/types/Content/Implementation.js

```diff
@@ -227,6 +227,21 @@ export class Content extends Relationship {
       blockTypes: this.blocks.map(block => (Array.isArray(block) ? block[0] : block).type),
 
       // Key the block options by type to be serialised and passed to the client
+      blockOptions: this.blocks
+        .map(block =>
+          Array.isArray(block)
+            ? [block[0], block[0].extendAdminMeta(block[1])]
+            : [block, block.extendAdminMeta({})]
+        )
+        .filter(([, blockConfig]) => blockConfig && Object.keys(blockConfig).length)
+        .reduce(
+          (options, block) => ({
+            ...options,
+            [block[0].type]: block[1],
+          }),
+          {}
+        ),
+
       blockOptions: this.blocks
```

Comment:
This should be removed right? Otherwise the blockOptions above would have no effect?

Reply:
🤦‍♂ Copy pasta error. Thanks!